pageserver/src/tenant/storage_layer/layer.rs

```diff
@@ -1278,7 +1282,7 @@ impl DownloadedLayer {
                     // TODO(#5815): we are not logging all errors, so temporarily log them **once**
                     // here as well
                     let permanent = permanent.context("load layer");
-                    tracing::error!("layer loading failed permanently: {permanent:#}");
+                    tracing::error!("layer loading failed permanently for {owner}: {permanent:#}");
```

Comment:
better use structured logging

```suggestion
                    tracing::error!(layer=%owner, "layer loading failed permanently: {permanent:#}");
```

Reply:
reverted anyway